src/js/components/DataSearch/__tests__/__snapshots__/DataSearch-test.tsx.snap

```diff
@@ -522,8 +522,8 @@ exports[`DataSearch renders 1`] = `
 .c7 {
   margin-left: 0px;
   margin-right: 0px;
-  margin-top: 6px;
-  margin-bottom: 6px;
+  margin-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below